### PR TITLE
doc: Modify the it8xxx2_evb overview document

### PR DESCRIPTION
**Pull request by hunterhu3000**
1. Remove the obsoleted board
2. Add HW rework of the new it8xxx2_evb
3. Add Wiring of the new it8xxx2_evb
4. Update the Ubuntu flash guide

Signed-off-by: Hu Zhenyu <zhenyu.hu@intel.com>

### CODE REVIEW

File: boards/riscv/it8xxx2_evb/doc/index.rst [hunk 1]
```diff
@@ -73,6 +75,41 @@ Other hardware features are not currently supported by Zephyr.
 The default configuration can be found in the
 :zephyr_file:`boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig` Kconfig file.
 
+HW reworks
```
Comment (keith-zephyr) [inline boards/riscv/it8xxx2_evb/doc/index.rst:78]: Nit - spell out "hardware"

File: boards/riscv/it8xxx2_evb/doc/index.rst [hunk 2]
```diff
@@ -73,6 +75,41 @@ Other hardware features are not currently supported by Zephyr.
...
+**********
+
+Before using the it8xxx2_evb, some HW rework is needed:
```
Comment (keith-zephyr) [inline boards/riscv/it8xxx2_evb/doc/index.rst:81]: same here - hardware

File: boards/riscv/it8xxx2_evb/doc/index.rst [hunk 3]
```diff
@@ -196,7 +225,10 @@ Ubuntu
       $ minicom -D /dev/ttyUSB0 -b 115200
 
 #. Open a second terminal window and use linux flash tool to flash your board.
-   You can find more details about tool usage from its document.
+
+   .. code-block:: console
+
+      $ sudo ~/itetool/ite -f build/zephyr/zephyr.bin
```
Comment (keith-zephyr) [inline boards/riscv/it8xxx2_evb/doc/index.rst:231]: Can add a new step 1 that instructs the user to download and compile the IT81202 Linux Flash Tool?

**Comment by hunterhu3000** [comment]
@keith-zephyr Thanks for your comments. I will update the document.
@GTLin08 Is there a link in the ITE official website to download the itedlb4-linux version 106 or later? Thanks

**Comment by Dino-Li** [comment]
> GTLin08 Is there a link in the ITE official website to download the itedlb4-linux version 106 or later? Thanks

@hunterhu3000 The flash tool can be downloaded from here:
https://www.ite.com.tw/uploads/product_download/itedlb4-linux-v106.tar.bz2

**Review by stephanosio — DISMISSED** [review]
There are some very large images added in this PR.

Please reduce the resolution and/or quality of the images so that are more reasonably sized (at most 100KB per image).

**Comment by hunterhu3000** [comment]
> There are some very large images added in this PR.
> 
> Please reduce the resolution and/or quality of the images so that are more reasonably sized (at most 100KB per image).

OK, will update it

**Review by keith-zephyr — DISMISSED** [review]

File: boards/riscv/it8xxx2_evb/doc/index.rst (hunk 3 above)
Comment (hunterhu3000) [reply boards/riscv/it8xxx2_evb/doc/index.rst:231]: The tar file contains a README file which instructs the user to compile and run the tool

**Comment by hunterhu3000** [comment]
@Dino-Li @GTLin08  could you please review and comment? Thanks

**Review by GTLin08 — DISMISSED** [review]

**Comment by fabiobaltieri** [comment]
Hey do we really want such an extensive list of rework instruction to be hosted on the main code repository? I can understand some basic board configuration (close/open a bridge, add/remove a jumper...) but this feels like something that should be part of some board documentation hosted on the vendor site. Imagine having to update these and add more revisions and variations. Wouldn't it make more sense to point at an external document with the rework instructions?

**Comment by NagarajuGundimi** [comment]
Most of the changes(rework instructions) looks good to me.

**Review by gmarull — DISMISSED** [review]
-1 to discuss whether rework instructions should be in Zephyr docs or in a vendor document.

**Comment by hunterhu3000** [comment]
@gmarull @fabiobaltieri The HW rework content/pictures are removed. Thanks to review.

File: boards/riscv/it8xxx2_evb/doc/index.rst [hunk 4]
```diff
@@ -8,9 +8,11 @@ Overview
 
 The IT8XXX2 is a 32-bit RISC-V Micro-controller.
 And a highly integrated embedded controller with system functions.
-It is suitable for mobile system applications.
+It is suitable for mobile system applications. The picture below is
+the INTEL_IT81302_81202_Colay_MECC_V01 develop board (also known as
```
Comment (fkokosinski) [inline boards/riscv/it8xxx2_evb/doc/index.rst:12]: This name looks weird. Can we spell it without underscores and in a more more human friendly way? Also EVB suggest evaluation board, not development board.

File: boards/riscv/it8xxx2_evb/doc/index.rst [hunk 5]
```diff
@@ -73,6 +75,12 @@ Other hardware features are not currently supported by Zephyr.
 The default configuration can be found in the
 :zephyr_file:`boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig` Kconfig file.
 
+Hardware reworks
+****************
+
+Before using the it8xxx2_evb, some hardware rework is needed. It is suggested
+to ask the board vendor for the rework guide.
```
Comment (fkokosinski) [inline boards/riscv/it8xxx2_evb/doc/index.rst:82]: IIRC, this is were the long and detailed HW rework guide was before. Is it available somewhere on the vendor site? If so, can we link it here?

Comment (hunterhu3000) [reply boards/riscv/it8xxx2_evb/doc/index.rst:82]: @Dino-Li @GTLin08  is it possible to upload a rework document somewhere, eg on ITE web site?

File: boards/riscv/it8xxx2_evb/doc/index.rst (hunk 4 above)
Comment (hunterhu3000) [reply boards/riscv/it8xxx2_evb/doc/index.rst:12]: OK, the name INTEL_IT81302_81202_Colay_MECC_V01 is printed on the board. We can call it IT81302 MECC board.

File: boards/riscv/it8xxx2_evb/doc/index.rst (hunk 5 above)
Comment (Dino-Li) [reply boards/riscv/it8xxx2_evb/doc/index.rst:82]: @hunterhu3000 sure, we will get back to you once it is ready.

Comment (Dino-Li) [reply boards/riscv/it8xxx2_evb/doc/index.rst:82]: @hunterhu3000 the rework document has been uploaded here:
https://www.ite.com.tw/uploads/product_download/IT81302_MECC_Rework_Guide_0927.pdf

Comment (hunterhu3000) [reply boards/riscv/it8xxx2_evb/doc/index.rst]: @fkokosinski all the change requests are updated. Thanks to review and +1

**Review by fkokosinski — APPROVED** [review]

**Review by enjiamai — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Pull request merged**